src/ripple/test/jtx/Env.h

```diff
@@ -60,38 +69,37 @@ namespace detail {
 // list or non-static data member initializer is
 // not implemented
 //
-template <std::size_t N>
-struct noripple_helper
+template <class T, std::size_t N>
+struct vararg_array_helper
 {
-    std::array<Account, N> args;
+    std::array<T, N> args;
 
-    template <class T, class ...Args>
-    std::array<Account, sizeof...(Args)>
+    template <class U, class ...Args>
+    std::array<T, sizeof...(Args)>
     flatten (Args&& ...args)
     {
-        return std::array<T,
-            sizeof...(Args)> {
-                std::forward<Args>(args)...};
+        return std::array<U,sizeof...(Args)> {
+            std::forward<Args>(args)...};
```

Comment:
I'm not sure what's going on in this workaround. `U` should always be the same as `T` and `sizeof...(Args)` should always be the same as `N`. I'd like to get rid of this complexity and replace `vararg_array_helper` with just `std:array`. We can use a type alias or just replace the current uses with `std::array`. Here's a branch that shows what I have in mind: https://github.com/ripple/rippled/commit/6281e3be09a310fc2625a7f0bddc5d1e69990698